test/Microsoft.Azure.WebJobs.Extensions.OpenApi.Core.Tests/Visitors/DictionaryObjectTypeVisitorTests.cs

```diff
@@ -69,6 +72,7 @@ public void Given_Type_When_IsParameterVisitable_Invoked_Then_It_Should_Return_R
         }
 
         [DataTestMethod]
+        [DataRow(typeof(Dictionary<object, string[]>), true)]
```

Comment:
In the OpenAPI spec, the Dictionary type is always Dictionary<string, ....>. It can't be Dictionary<object, ...>.